### PR TITLE
Prepare transcriptome indices for 2019 Houston training workshop

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Related: https://github.com/AlexsLemonade/training-modules/issues/37

Here, I'm generating Salmon transcriptome indices of both lengths and their corresponding tx2gene mappings for the following organisms:
- _Homo sapiens_
- _Mus musculus_
- _Danio rerio_
- _Canis familiaris_

The line count looks pretty brutal, but that is because I've added the tx2gene TSV files here as they are pretty small.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# training-txome-prep
\ No newline at end of file
+# Salmon transcriptome index preparation
+
+This repository is intended to document how Salmon transcriptome indices are generated for RNA-seq and scRNA-seq training modules.
+Currently, this corresponds to our 2019 Houston workshop.
+
+We run all scripts in the Docker container for training (introduced here: [AlexsLemonade/RNA-Seq-Exercises](https://github.com/AlexsLemonade/RNA-Seq-Exercises/pull/22)).
+_Note: `optparse` is not installed on that Docker image but is required to generate the tx2gene files._
```
Comment (cansavvy) [inline README.md:7]: Just to clarify, does this mean if you are to run this in the container, you need to install optparse before you run it?

Comment (cansavvy) [reply README.md:7]: If so, should you just add a line in `scripts/get_tx2gene.R` that installs optparse if it is not installed?

**Review by cansavvy — APPROVED** [review]
LGTM. My only question is about `optparse` and it's installation.

File: README.md (hunk 1 above)
Comment (jaclyn-taroni) [reply README.md:7]: You are correct. This sounds like a good idea.

**Pull request merged**